src/EditorFeatures/CSharp/Outlining/Outliners/DocumentationCommentOutliner.cs

```diff
@@ -35,12 +36,39 @@ private static string GetBannerText(DocumentationCommentTriviaSyntax documentati
             string text;
             if (summaryElement != null)
             {
-                var summaryText = from xmlText in summaryElement.ChildNodes().OfType<XmlTextSyntax>()
-                                  from tk in xmlText.TextTokens
-                                  let s = tk.ToString().Trim()
-                                  where s.Length > 0
-                                  select s;
-                text = prefix + " <summary> " + string.Join(" ", summaryText);
+                var sb = new StringBuilder(summaryElement.Span.Length);
+                sb.Append(prefix);
+                sb.Append(" <summary>");
+                foreach (var node in summaryElement.ChildNodes())
+                {
+                    if (node is XmlTextSyntax)
+                    {
+                        foreach (var tk in ((XmlTextSyntax)node).TextTokens)
+                        {
+                            var s = tk.ToString().Trim();
+                            if (s.Length > 0)
+                            {
+                                sb.Append(" ");
+                                sb.Append(s);
+                            }
+                        }
+                    }
+                    else if (node is XmlEmptyElementSyntax)
+                    {
+                        var e = (XmlEmptyElementSyntax)node;
+                        if (e.Name.LocalName.ToString() == "see")
+                        {
+                            var cref = e.Attributes.OfType<XmlCrefAttributeSyntax>().FirstOrDefault();
+                            if (cref != null)
+                            {
+                                sb.Append(" ");
+                                sb.Append(cref.Cref.ToString());
```

Comment:
we don't have to do any extra processing for symbols like generics or constructor that contains backticks or #ctor etc. correct?